app/src/features/apiClient/screens/apiClient/components/clientView/APIClientView.tsx

```diff
@@ -423,7 +428,11 @@ const APIClientView: React.FC<Props> = ({ apiEntry, apiEntryDetails, notifyApiRe
       );
       setEntry({ ...result.data.data, response: entry.response, testResults: entry.testResults });
       resetChanges();
-      trackRequestSaved("api_client_view");
+      trackRequestSaved({
+        src: "api_client_view",
+        has_scripts: Boolean(entry.scripts?.preRequest?.length),
```

Comment:
```suggestion
        has_scripts: Boolean(entry.scripts?.preRequest),
```
Shouldn't this be enough?